external-crates/move/move-execution/v0/crates/move-vm-runtime/src/data_cache.rs

```diff
@@ -126,6 +126,26 @@ impl<S: MoveResolver> DataStore for TransactionDataCache<S> {
         }
     }
 
+    fn load_package(&self, package_id: &AccountAddress) -> VMResult<Vec<Vec<u8>>> {
```

Comment:
I would have hoped version cuts could have avoided needing this sort of stuff, but I see we are sort of stuck where we are. Any solution I have isn't particularly better. Maybe on a longer timeline the version cut should become the whole `crates/` folder...

Reply:
Yea, versioning the whole thing is the only way around this :'( 

That being said though I realized I can just stub these out -- they should never be accessed from old versions.